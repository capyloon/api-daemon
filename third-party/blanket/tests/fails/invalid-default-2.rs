extern crate blanket;
use blanket::blanket;

#[blanket(default = "a b")]
pub trait MyTrait {}

pub fn main() {}
