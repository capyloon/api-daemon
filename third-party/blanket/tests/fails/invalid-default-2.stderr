error: expected module identifier
 --> $DIR/invalid-default-2.rs:4:21
  |
4 | #[blanket(default = "a b")]
  |                     ^^^^^
